src/main/java/br/com/mercadolivre/loja/user/UserForm.java

```diff
@@ -0,0 +1,36 @@
+package br.com.mercadolivre.loja.user;
+
+import org.hibernate.validator.constraints.Length;
+
+import javax.validation.constraints.*;
+
+public class UserForm {
+
+    @NotNull
+    @Email
+    private String email;
+
+    @NotNull
```

Comment:
A senha não deveria estar em branco:

```suggestion
    @NotBlank
```